unoplat-code-confluence-commons/business_logic_references.md

```
@@ -1,27 +1,1609 @@
+        le=1.0,
+        description="Baseline confidence for CallExpression feature definitions",
+    )
+    startpoint: bool = Field(
+        default=False,
+        description="Indicates whether this feature represents a starting point or entry point in the application",
+    )
+
+    @property
+    def feature_key(self) -> str:
+        """Return the dotted convenience key derived from structured identity."""
+        return _compose_feature_key(self.capability_key, self.operation_key)
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)